pkg/router/routes/api/job.go

```diff
@@ -53,11 +59,16 @@ func CreateJob(w http.ResponseWriter, r *http.Request) {
 }
 
 func UpdateJob(w http.ResponseWriter, r *http.Request) {
+	err := limit.IsOverLimit(r)
+	if err != nil {
+		w.WriteHeader(http.StatusTooManyRequests)
+		return
+	}
 	if r.Method != http.MethodPost {
 		w.WriteHeader(http.StatusMethodNotAllowed)
 		return
 	}
```

Comment:
You can merge the two if statements and save the duplicate code

Reply:
Those are different codes